integration_tests/commands/async/object_test.go

```diff
@@ -25,6 +26,76 @@ func TestObjectCommand(t *testing.T) {
 			assertType: []string{"equal", "assert", "assert", "equal", "assert"},
 			delay:      []time.Duration{0, 2 * time.Second, 3 * time.Second, 0, 0},
 		},
+		{
+			name:       "Object Encoding check for raw",
+			commands:   []string{"SET foo foobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobar", "OBJECT ENCODING foo"},
+			expected:   []interface{}{"OK", "raw"},
+			assertType: []string{"equal", "equal"},
+			delay:      []time.Duration{0, 2 * time.Second},
+		},
+		{
+			name:       "Object Encoding check for int",
+			commands:   []string{"SET foo 1", "OBJECT ENCODING foo"},
+			expected:   []interface{}{"OK", "int"},
+			assertType: []string{"equal", "equal"},
+			delay:      []time.Duration{0, 2 * time.Second},
+		},
+		{
+			name:       "Object Encoding check for embstr",
+			commands:   []string{"SET foo bar", "OBJECT ENCODING foo"},
+			expected:   []interface{}{"OK", "embstr"},
+			assertType: []string{"equal", "equal"},
+			delay:      []time.Duration{0, 2 * time.Second},
+		},
+		{
+			name:       "Object Encoding check for deque",
+			commands:   []string{"LPUSH listKey 'value1'", "LPUSH listKey 'value2'", "OBJECT ENCODING listKey"},
+			expected:   []interface{}{int64(1), int64(2), "deque"},
+			assertType: []string{"assert", "assert", "equal"},
+			delay:      []time.Duration{0, 2 * time.Second, 3 * time.Second},
+		},
+		{
+			name:       "Object Encoding check for bf",
+			commands:   []string{"BFADD bloomkey value1", "BFADD bloomkey value2", "OBJECT ENCODING bloomkey"},
+			expected:   []interface{}{int64(1), int64(1), "bf"},
+			assertType: []string{"assert", "assert", "equal"},
+			delay:      []time.Duration{0, 2 * time.Second, 3 * time.Second},
```

Comment:
Hi @sashpawar11 , Could you please share why we are testing the cases with delay here?

Reply:
Hi @apoorvyadav1111 - assumed to keep a delay for the bloom filter to be created based on other tests. Please advise if it is unnecessary, I will go ahead and remove the delay once confirmed.

Thankyou.

Reply:
Hey, if you think it is needed for a particular test case, please keep the delay. But lets keep the value 0 for the commands which are not using it or doesn't affect the functionality as it increases the runtime for tests.
Thanks

Reply:
@apoorvyadav1111 - sure, resolved the merge conflict for now. Will recommit with the mentioned changes soon.
Thanks for your time!

Reply:
Hey @apoorvyadav1111 - pushed the latest fix and the tests + build have passed. Please review the same, thankyou!